kubernetes/laravel-echo-server.deployment.yml

```diff
@@ -9,6 +9,8 @@ spec:
     matchLabels:
       name: laravel-echo-server
   replicas: 1
+  lookupPolicy:
+    local: true
```

Comment:
There is no such field in the spec https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#deploymentspec-v1-apps. You mixed it up with Openshift Image Streams.